src/pages/home/report/ReportActionCompose.js

```diff
@@ -536,17 +649,18 @@ class ReportActionCompose extends React.Component {
         const inputPlaceholder = this.getInputPlaceholder();
         const encodedCommentLength = ReportUtils.getCommentLength(this.comment);
         const hasExceededMaxCommentLength = encodedCommentLength > CONST.MAX_COMMENT_LENGTH;
+        const isUseFocusedColor = !isBlockedFromConcierge
+          && !this.props.disabled
```

Comment:
```suggestion
            && !this.props.disabled
```

Reply:
Surprised the linter didn't catch this. Please be sure to use 4-space indentation.